english_to_arabic_transphonator/transphonator.py

```diff
@@ -39,6 +39,9 @@ def _find_equivalent_character(self, phoneme):
         )
         return self.transliteration_map[available_phonemes[max_idx]]
 
+    def transphonate_english_sentence(self, english_sentance):
+        return " ".join([self.transphonate_english_word(word) for word in english_sentance.split()])
```

Comment:
Thanks for the contribution, Mohamed!
This is much appreciated, and I would happily discuss your ideas 🙂 
I personally do not think whitespace tokenization is enough, so I would go with something like spacy's tokenizers (https://spacy.io/api/tokenizer#call).
Moreover, I do not want to have spacy as a dependency for this package just to make it lightweight.

Reply:
So, it's better to leave the choice of the tokenizer to the users instead of taking this responsibility.

Reply:
I see, this is a valid point (i wished to contribute the project ^^)